Add documentation for option out of layout with layout={false} and fix Signal Overview example (#1185)
motion-canvas/motion-canvas: packages/docs/docs/getting-started/layouts.mdx

<ApiSnippet url={'/api/2d/components/Layout#offset'} />
<hr />

> ## Opting Out of Layout 

Reviewer: This should be in sentence case, like the other sections in the file.
```suggestion
## Opting out of layout
```

> ## Opting Out of Layout 

Reviewer: - [ ] This should also probably be listed as a property, i.e. `Layout.layout` and stay here, or should be moved above the property listing.

Author: I am confused property listing 

Do you mean I should mention Layout.layout as api reference in docs too ?

You can disable the layout behaviour by passing `layout={false}` to your component:

```tsx
<Layout layout direction="row">
  <Rect width={100} height={100} fill="green"/>
  <Rect width={100} height={100} fill="red" layout={false} x={200} y={50}/>
</Layout>
```

> This behaves almost similar to **absolute postioning** in CSS Flexbox: 

Reviewer: "almost similar" is redundant, and we don't frequently use bold like this. I'd recommend either making it a hyperlink to the MDN docs, or removing it. Also, positioning is misspelled.

```suggestion
This behaves like absolute positioning in CSS Flexbox: 
```

> - The node doesn't affect or get affected by layout flow

Reviewer: ```suggestion
- The node doesn't affect or get affected by layout flow
- The node doesn't lay out its own children
```

- You're responsible for positioning it manually
- It can still be visually inside a layout container

```css

> .parent{

Reviewer: ```suggestion
.parent {
```

>   display:flex;

Reviewer: ```suggestion
  display: flex;
```

}

> .child{

Reviewer: ```suggestion
.child {
```

  position: absolute;
}
```

> You can learn more about such layout exceptions in the [Groups](#groups) section below.

Reviewer: Space between sections. Plus a small tweak for concision.
```suggestion
You can learn more about layout exceptions in the [Groups](#groups) section below.

```

As an aside, it might make sense to group the Group section and this one together so that one looking for the exceptions can find both at once.

<ApiSnippet url={'/api/2d/components/Layout#offset'} />
<hr />

## Opting out of layout 

> You can disable the layout behaviour by passing `layout={false}` to your component:

Reviewer: nit: consistency (docs use behavior in other places)
```suggestion
You can disable layout behavior by passing `layout={false}` to your component:
```

```tsx
<Layout layout direction="row">
  <Rect width={100} height={100} fill="green"/>
  <Rect width={100} height={100} fill="red" layout={false} x={200} y={50}/>
</Layout>
```

This behaves like absolute postioning in CSS Flexbox: 
- The node doesn't affect or get affected by layout flow
- The node doesn't lay out its own children
- You're responsible for positioning it manually
- It can still be visually inside a layout container

```css
.parent {
  display: flex;
}

.child {
  position: absolute;
}
```

> ### `Layout.layout`

Reviewer: To be clear about this, you're adding this guide-type section in the middle of a bunch of reference-type API snippets. I don't think we need the `ApiSnippet`, but I do think that this should belong with the other guide sections above.

I think the Groups section has a similar problem, and it should be up closer to the top with the guide-type sections. Maybe this is more a change for when I eventually overhaul the docs (which I've been working on, slowly).

Author: I moved the group section and applied the changes
